Source: termux-packages c12cdeaa03dc292a0d0703b0b87082ccc41eae23, packages/mesa/0000-disable-android-detection.patch
Why: the imagefs is a Linux-like userland (X11), not an Android graphics stack; Mesa must not take its Android
paths (liblog, gralloc, cutils). Ludashi's build has the same change (no liblog, dlopens libvulkan.so.1).

--- a/src/util/detect_os.h
+++ b/src/util/detect_os.h
@@ -24,7 +24,7 @@
  * Android defines __linux__, so DETECT_OS_LINUX and DETECT_OS_POSIX will
  * also be defined.
  */
-#if defined(__ANDROID__)
+#if 0
 #define DETECT_OS_ANDROID 1
 #endif

--- a/include/vulkan/vk_android_native_buffer.h
+++ b/include/vulkan/vk_android_native_buffer.h
@@ -17,7 +17,7 @@
  * system rather the build target.
  */
 
-#if defined(__ANDROID__) || defined(ANDROID)
+#if 0
 
 #include <cutils/native_handle.h>
 #if ANDROID_API_LEVEL < 28
